src/main/scala/org/clulab/taxero/TaxonomyReader.scala

```diff
@@ -49,38 +49,38 @@ class TaxonomyReader(
 
   val proc = new FastNLPProcessor
 
-  def getRankedHypernyms(tokens: Seq[String]): Seq[ScoredMatch] = {
-    val lemmas = convertToLemmas(tokens)
-    val extractors = mkHypernymExtractors(lemmas)
+  def getRankedHypernyms(tokens: Seq[String], lemmatize: Boolean): Seq[ScoredMatch] = {
+    val items = if (lemmatize==true) convertToLemmas(tokens) else tokens
+    val extractors = mkHypernymExtractors(items, lemmatize)
     val matches = getMatches(extractors)
-    rankMatches(lemmas, matches)
+    rankMatches(items, matches)
   }
 
-  def getRankedHyponyms(tokens: Seq[String]): Seq[ScoredMatch] = {
-    val lemmas = convertToLemmas(tokens)
-    val extractors = mkHyponymExtractors(lemmas)
+  def getRankedHyponyms(tokens: Seq[String], lemmatize: Boolean): Seq[ScoredMatch] = {
+    val items = if (lemmatize==true) convertToLemmas(tokens) else tokens
+    val extractors = mkHyponymExtractors(items, lemmatize)
     val matches = getMatches(extractors)
-    rankMatches(lemmas, matches)
+    rankMatches(items, matches)
   }
 
-  def getRankedCohyponyms(tokens: Seq[String]): Seq[ScoredMatch] = {
-    val lemmas = convertToLemmas(tokens)
-    val extractors = mkCohyponymExtractors(lemmas)
+  def getRankedCohyponyms(tokens: Seq[String], lemmatize: Boolean): Seq[ScoredMatch] = {
+    val items = if (lemmatize==true) convertToLemmas(tokens) else tokens
+    val extractors = mkCohyponymExtractors(items, lemmatize)
     val matches = getMatches(extractors)
-    rankMatches(lemmas, matches) 
+    rankMatches(items, matches)
   }
 
   def getExpandedHypernyms(pattern: Seq[String], n: Int): Seq[ScoredMatch] = {
     // start query set with the provided query
     val allQueries = mutable.HashSet(pattern)
     // add the n closest cohyponyms to the query set
-    allQueries ++= getRankedCohyponyms(pattern).take(n).map(_.result)
+    allQueries ++= getRankedCohyponyms(pattern, lemmatize = true).take(n).map(_.result)
```

Comment:
in all these places where you have `lemmatize=true`, we should prob set one variable above, that is loaded from a config and use it in these places, to not adjust existing functionality of the webapp

Reply:
hmmm -- though I'm ok with the just saying `=true` in the HomeController, so maybe that's fine, but why are you enforcing it here -- this is the only place, right?